src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -505,12 +505,38 @@ export async function getFeatureIdType(output: Log, env: NodeJS.ProcessEnv, id:
 	return { type: 'github-repo', manifest: undefined };
 }
 
+export function getBackwardCompatibleFeatureId(id: string) {
```

Comment:
💀 Should not be merged until we are ready for open sourcing /images and /features repositories